src/pages/Adopt.js

```diff
@@ -1,13 +1,22 @@
 import React from "react"
+import { useState } from "react"
+import ZipCodeInput from "../components/ZipCodeInput"
 import PetList from "../components/PetList"
+import { LoadingContext } from "../context"
+import LoadingMessage from "../components/LoadingMessage"
 import Search from "../components/Search"
 
+
 const Adopt = () => {
+  const [isLoading, setIsLoading] = React.useState(false);
   return (
-    <div>
-      <Search />
-      <PetList />
```

Comment:
These two lines are needed to have the <Search/> component that allows the user to select a zip code and pet type. <ZipCodeInput/> was moved to Search.js in a previous pull.